ui/apps/platform/src/services/AdministrationEventsService.ts

```diff
@@ -139,6 +139,24 @@ export function listAdministrationEvents(
         .then((response) => response.data.events);
 }
 
+type GetAdministrationEventResponseArg = {
```

Comment:
nit: putting "Response" at the end of the type name would emphasize that it is the return value of that function

```suggestion
type GetAdministrationEventArgResponse = {
```